src/style.css

```diff
@@ -138,3 +138,48 @@ color:white !important;
   background-position: center;
   right:0;
 }
+ /*login part  */
+@font-face{
+  font-family: "Reader Trial-Regular";
+  font-style: normal;
+  font-weight: 400;
+  src: url("https://fonts.animaapp.com/ReaderTrial-Regular") format("opentype");
```

Comment:
Create a CSS variable for URL value, and are we using more than one font?

Reply:
update: imported global font present locally